doc/user/interop.md

```diff
@@ -85,8 +95,33 @@ Otherwise, return the instance variable names, without the leading `@`.
 
 `KEYS(other)` → `other.instance_variables.map { |key| ...key without @... }`
 
-In both cases the keys are returned as a Ruby `Array` containing Ruby `String`
-objects.
+In both cases if the `internal` flag is set, instance variable names with the
```

Comment:
This sounds strange to me. Why would hash keys have an '@' prepended?

Reply:
Should it be cased as `INTERNAL` to match other usages in the document?

Reply:
Reworded. `internal` is a parameter, not the message `INTERNAL`.